backend/src/v2/component/launcher_v2.go

```diff
@@ -472,47 +522,49 @@ func uploadOutputArtifacts(ctx context.Context, executorInput *pipelinespec.Exec
 		if len(artifactList.Artifacts) == 0 {
 			continue
 		}
-		// TODO: Support multiple artifacts someday, probably through the v2 engine.
-		outputArtifact := artifactList.Artifacts[0]
 
-		// Merge executor output artifact info with executor input
-		if list, ok := executorOutput.Artifacts[name]; ok && len(list.Artifacts) > 0 {
-			mergeRuntimeArtifacts(list.Artifacts[0], outputArtifact)
-		}
+		for _, outputArtifact := range artifactList.Artifacts {
+			glog.Infof("outputArtifact in uploadOutputArtifacts call: ", outputArtifact)
```

Comment:
```suggestion
			glog.Infof("outputArtifact in uploadOutputArtifacts call: %s", outputArtifact.Name)
```
did you mean to do something like this?

Reply:
I was logging the whole thing while validating / debugging, but I think `.Name` is sufficient going forward. Updated.